Dev

mark-solovyev opened the pull request.

mark-solovyev committed:
The week1 branch was created

mark-solovyev committed:
Created three section of the site as the technical task required

mark-solovyev committed:
Put the code that the wrapper in the index file contained into the section tag

mark-solovyev committed:
Put three files together

b-ars07 commented:
1. нет контейнера в секциях
2. секция отзывы должна быть на всю ширину сайта
3. для шрифтов нужно прописать свойства прописаны св-ва 
-webkit-font-smoothing: antialiased;
	-moz-osx-font-smoothing: grayscale;
4. формат шрифтов - woff и woff2
5. секции высотой должны быть 100vh и минмальная высота секций по макету
6. я на созвоне говорил про фиксированное меню и где оно должно лежать в верстке
7. ошибка БЭМ - нет класса
8. Меню лежит в теге nav и содержит список с пунктами меню
9. точки между пунктами меню это не элементы списка, а лучше их верстать псевдоэлементом :after
10. ошибки БЭМ, некорректные названия классов, не понятно что за блоки
11. тут не нужен паддинг, можно все сделать через флексы) - http://joxi.ru/5md34MphkXazPr
12. толкать лучше сверху вних, т.е. 1 элемент толкает второй. и через margin-bottom - http://joxi.ru/1A5adpeTnNzjYA
13. зачем тут это? - http://joxi.ru/l2ZQb68Iw4RvPm
14. это не новый список, это элемент списка - http://joxi.ru/eAONB1aUxY6XB2
15. я на созвоне рассказывал как стилизовать эти блоки и почему им не нужен класс - http://joxi.ru/4Ak95ewhy60qdm
16. Содержимое секций выравнивается по центру (за исключением секции с отзывами)
17. списком делаются только идентичные по верстке и визуальному изображению блоки, это же совсем разные блоки, их нельзя засовывать в список - http://joxi.ru/J2bablETXZgKwm

mark-solovyev committed:
I sorted out the fonts

mark-solovyev committed:
Temporary commit

mark-solovyev committed:
Исправил почти всё — кроме плавающего меню

mark-solovyev committed:
Fixed the float menu css

b-ars07 commented:
1. это что такое? - http://joxi.ru/eAONB1aUxLG4a2
2. контейнер у тебя появился, но он не ограничивает контент. Ты смотрел наш групповой созвон?
3. фон не нужно задавать контейнеру, а класс для секции свой - http://joxi.ru/Vm6RL7ecDeK41m
4. минимальная высота не правильная должна быть 650vh
5. это не футер - http://joxi.ru/4Ak95ewhyOV18m
6. ошибка БЭМ что у li разные классы, точки это не элементы списка - http://joxi.ru/ZrJ0JyWc9EkWG2
7. НЕ ИСПРАВИЛ для точек лучше использовать after - http://joxi.ru/ZrJ0JyWc9EkWG2
8. это не padding, это логотип толкает блок с меню, т.е. margin-right у логотипа! - http://joxi.ru/EA4ZWveCwe0Mjm
9. куда ведет эта ссылка? - http://joxi.ru/MAj9QlVh4eY8x2
10. что это за свойства?? спозиционируй все на флексах)) не нужно высоту 96%))) - http://joxi.ru/l2ZQb68Iw9Vebm
11. не надо разбивать на 2 тега цельный текст + это не заголовок второго уровня - http://joxi.ru/nAyMk83IYokVaA
12. НЕ ИСПРАВИЛ контент нужно центрировать по вертикали - http://joxi.ru/xAeOb0lTpQV7Gm
13. названия классов не по БЭМ, абсолютно ен понятные, что лежит в этих блоках - http://joxi.ru/brRpJ6OtJj8M02
14. НЕ ИСПРАВИЛ, я просил не делать тут отступами, а спозиционируй флексами - http://joxi.ru/J2bablETXQMlWm
15. почему колонка? - http://joxi.ru/Q2KBl7dT4Dgagm
16. откуда эти значения?? тк как 4 плитки в ряд, т.е. 100% / 4 = 25% ширина, и в высоту 2 плитки, значит 100% / 2 = 50% высота

mark-solovyev committed:
Исправил пункты 6 и 7

mark-solovyev committed:
Исправил пункты 8 — 12

Хотя, всё равно есть вопросы по пункту 12: не нравится как выглядит
центрирование по-вертикали; смотрелось бы лучше, если бы контент был бы
смещён чуть-чуть выше.
По пункту 9: а куда должна вести ссылка? Никуда не ведёт — страница-то,
на которую она должна вести, ещё не свёрстана.

mark-solovyev committed:
Пункты 13 и 14

13. Названия классов исправил
14. Отступы убрал. Но если у нас есть два элемента, выравненные по центру flex'ом, и я хочу чуть-чуть отделить их друг от друга — как это ещё сделать, если не отступами?!

mark-solovyev committed:
Исправил оставшиеся пункты

Пункт 4: «минимальная высота неправильная, должна быть 650vh» — не
понял, высота ЧЕГО должна быть 650vh ?
Пункт 15: — «Почему колонка?» – Потому что контейнеры первых двух секций состоят из двух элементов, которые надо расположить сверху вниз
